plugins/jenkins-backend/src/service/jenkinsApi.ts

```diff
@@ -30,6 +30,12 @@ import {
 import { jenkinsExecutePermission } from '@backstage/plugin-jenkins-common';
 import { NotAllowedError } from '@backstage/errors';
 
+if (!Promise.any) {
```

Comment:
could also have done this inside getProjects where you're already in an async function

Reply:
That's a good point.